### PR TITLE
Enable test on github actions

### PR DESCRIPTION
**Pull request by wenyingd**
1. Add github action to run test inside container.
2. Fix issues in the existing test code.

Signed-off-by: wenyingd <wenyingd@vmware.com>

### CODE REVIEW

File: build/images/test/test-integration [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+trap "chown -R $1:$2 /tmp/gopath /tmp/gocache" EXIT
+
+/usr/share/openvswitch/scripts/ovs-ctl --system-id=random start --db-file=/var/run/openvswitch/conf.db
+ovs-vsctl set-manager ptcp:6640
+make test-integration
```
[the remark this answers is not in the document]
Comment (wenyingd) [reply build/images/test/test-integration]: udpated.

File: Makefile [hunk 2]
```diff
@@ -20,3 +31,31 @@ golangci: .golangci-bin
...
+	@docker run --privileged --rm \
+		-e "GOCACHE=/tmp/gocache" \
+		-e "GOPATH=/tmp/gopath" \
+		-e "INCONTAINER=true" \
+		-w /usr/src/antrea.io/ofnet \
+		-v $(DOCKER_CACHE)/gopath:/tmp/gopath \
+		-v $(DOCKER_CACHE)/gocache:/tmp/gocache \
+		-v $(CURDIR):/usr/src/antrea.io/ofnet:ro \
+		-v /lib/modules:/lib/modules \
+		--sysctl net.ipv6.conf.all.disable_ipv6=0 \
```
[the remark this answers is not in the document]
Comment (wenyingd) [reply Makefile]: removed

File: ofctrl/ofpacket_test.go [hunk 3]
```diff
@@ -260,6 +277,7 @@ func generateTCPPacketOut(srcMAC, dstMAC net.HardwareAddr, srcIP net.IP, dstIP n
 }
 
 // keeping this in case it is useful later
+//
```
[the remark this answers is not in the document]
Comment (wenyingd) [reply ofctrl/ofpacket_test.go]: It is updated by the IDE automatically, updated.

**Review by ashish-varma — APPROVED** [review]
Looks good to me.

**Pull request merged**